index.js

```diff
@@ -0,0 +1,68 @@
+import Book from './js/bclass.js';
+import { DateTime } from './node_modules/luxon/build/es6/luxon.js';
+
```

Comment:
- [ ] WELL DONE 👍. You have successfully installed and imported a package called `luxon`. However, you forgot to use it after installing 😜. Please firstly see [Get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time) and add the date to your footer like this image 🍀.

Example
![image](https://user-images.githubusercontent.com/69011963/148747267-87cb5880-a3ac-4560-b030-c70a311ba5ff.png)